pkg/controller/provider/controller.go

```diff
@@ -190,6 +190,31 @@ func (r Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (r
 		}
 	}
 
+	if provider.DeletionTimestamp != nil && k8sutil.ContainsFinalizer(provider, api.OvaProviderFinalizer) {
+		fmt.Println("we are here at finalizer")
```

Comment:
forgot to delete?

Reply:
yes, looks like it was used for debugging, @bkhizgiy  please drop